pandora-server-directory/src/spaces/space.ts

```diff
@@ -360,8 +391,14 @@ export class Space {
 			return 'ok';
 
 		// If the space is full, you cannot enter it (some checks ignore space being full)
-		if (this.characterCount >= this.config.maxUsers && !ignore.characterLimit)
-			return 'errFull';
+		if (!ignore.characterLimit) {
+			let maxUsers = this.config.maxUsers;
+			if (this.isAdmin(character.baseInfo.account)) {
+				maxUsers = Math.ceil(maxUsers * LIMIT_SPACE_MAX_CHARACTER_ADMIN_OVERSHOT_MULTIPLIER);
+			}
```

Comment:
I'm not sure if we want this overshoot to be for admins, or only for owners. While it is not configurable only owners makes more sense to me.
Would you also please create an issue that we want to make this configurable?

Reply:
I think that argument makes indeed sense.
With a static overshoot limit, there is a  risk an owner could not be allowed in, if there are many admins.
Making things more configurable later makes definitely sense.

Reply:
I made it a fixed 5 for owners

Reply:
also, would it make sense to limit the overshot to owners, since admins can do anything an owner could do
and to me this was to prevent a room from being filled with normal characters
like what if you fill a room with all inactive or characters who are unable to leave
both cases admin can fix the room just as much as an owner could
and considering we do not have shared ownership implemented it would make sense to allow overshot for admins and owners just the same

Reply:
Owners can do a bit more than admins:
- They can't be de-owned, de-admined or banned by others
- They can add more owners
So "hostile" admin (even singular) could abuse this to fill room they don't own with dummy characters until owner can't join and just leave it there. At that point the only thing owner could do would be to abandon the room from the search screen, which would mean loosing anything inside.